guide/C/ch_invest.xml

```diff
@@ -1404,30 +1398,29 @@ Income
             </listitem>
 
             <listitem>
-              <para>Locate the folder where &app; is installed by searching for <application>gnc-fq-update</application>
+              <para>Locate the folder where &app; is installed by searching for <filename>gnc-fq-update</filename>.
               </para>
             </listitem>
 
             <listitem>
-              <para>Change to that directory, open a root shell
+              <para>Change to that directory, open a root shell.
               </para>
             </listitem>
 
             <listitem>
-              <para>Run the command <command>gnc-fq-update</command>
+              <para>Run the command <command>./gnc-fq-update</command>.
               </para>
             </listitem>
           </itemizedlist>
           This will launch a <application>Perl</application> CPAN update session that will go out
-          onto the internet and install the <application>Finance::Quote</application> module on your
+          onto the internet and install the &app-fq; module on your
           system. The gnc-fq-update program is interactive, however, with most systems you should be
-          able to answer <quote>no</quote> to the first question: <quote>Are you ready for manual
-          configuration? [yes]</quote> and the update will continue automatically from that point.
+          able to answer <userinput>no</userinput> to the first question: <computeroutput>Are you ready for manual
+          configuration? [yes]</computeroutput>, and the update will continue automatically from that point.
```

Comment:
Because I'd like to split the sentence, I replace it with
`              able to answer the first question <computeroutput>Are you ready for manual
              configuration? [yes]</computeroutput> with <userinput>no</userinput>. Then the update will continue automatically from that point.
            </para>`

Long sentences with conjunctions hinder translation, especially to no-latin languages.

Reply:
Yes, feel free to do anything to make them easier to understand and to translate, but no space before `</para>`

Reply:
Done.